src/test/skript/tests/syntaxes/expressions/ExprItemsIn.sk

```diff
@@ -0,0 +1,19 @@
+test "filtering items in expression":
+	set {_world} to random world out of all worlds
+	set block at spawn of {_world} to chest
+	set {_inv} to inventory of (block at spawn of {_world})
+	set slot 1 of {_inv} to dirt
+	set slot 2 of {_inv} to stone
+	set slot 3 of {_inv} to bucket
+	assert all blocks in inventory {_inv} are dirt or stone with "found correct items with ExprItemsIn##get"
```

Comment:
Is this supposed to be `all items`?

Reply:
nope - this test is to test the filtering part by filtering for any blocks (which is why it checks for everything except bucket)